src/ProcessFailedException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace IndieHD\AudioManipulator;
+
+use Symfony\Component\Process\Exception\ProcessFailedException as SymfonyProcessFailedException;
+
+class ProcessFailedException extends SymfonyProcessFailedException
+{
+    public function __construct(Process $process)
```

Comment:
Might be a good idea to code to the `ProcessInterface` here ?

Reply:
Good catch; fixed in a subsequent commit.